arviz/plots/khatplot.py

```diff
@@ -136,13 +136,14 @@ def plot_khat(
     hlines_kwargs.setdefault("linestyle", [":", "-.", "--", "-"])
     hlines_kwargs.setdefault("alpha", 0.7)
     hlines_kwargs.setdefault("zorder", -1)
-    hlines_kwargs.setdefault("color", "C1")
+    hlines_kwargs.setdefault("color", to_hex("C1"))
 
     if coords is None:
         coords = {}
 
     if color is None:
         color = "C0"
+    color = to_hex(color)
```

Comment:
Ooohh, I see.  Plot khat accepts plenty of options, from a color string to a dimension name. From the [docs](https://arviz-devs.github.io/arviz/generated/arviz.plot_khat.html#arviz.plot_khat).

> color : str or array_like, optional
>
>    Colors of the scatter plot, if color is a str all dots will have the same color, if it is the size of the observations, each dot will have the specified color, otherwise, it will be interpreted as a list of the dims to be used for the color code

the conversion to hex should be done somewhere later in the code (after dim names have been converted to colors) and also take into account that colors can be arrays, we should probably define a function in plot_utils to take into account vectorized cases.

I didn't think about this before.

Reply:
Thanks for the clarification. I'll make the changes accordingly.

Reply:
I have been trying to make the changes for quite some time now but I am not quite getting them right. Just wanted to double-check: the Matplotlib color argument for to_rgba and to_hex is essentially a list right?